### DISCUSSION TITLE
Open-Set-Go Logo Design suggestion

### DISCUSSION THREAD

**Discussion by ymw0407** [AgainIoT/Open-Set-Go#48] [Feature Suggestion / Share Ideas]
Currently, Open-Set-Go's Logo seems to have the disadvantage of being transparent and invisible except for Logo as shown in the picture below! That's why I think you'll be able to see it better if you use white around the rim!
현재 Open-Set-Go의 Logo는 아래의 사진과 같이 Logo를 제외하고 모두 투명하여 잘 보이지 않는다는 단점이 있는 것 같습니다! 그렇기 때문에 테두리에 흰색을 원형으로 둘러서 사용하면 좀 더 잘 보일것 같습니다!
![image](https://github.com/AgainIoT/Open-Set-Go/assets/77202633/018d0131-eb51-41e0-bc64-60756c694afe)

Just like Google uses it like this!
구글이 이렇게 사용하는 것 처럼요!
![image](https://github.com/AgainIoT/Open-Set-Go/assets/77202633/aa0878b8-8bac-4ab8-8548-9952af683e9c)

**Comment by ymw0407** [comment]
모든 로고를 이렇게 사용할 필요는 없지만, favicon으로 뜨는 로고는 이렇게 진행되면 좋을 것 같습니다!
Not all logos need to be used like this, but logos that pop up as favicon would be nice to go like this!

**Comment by hummingbbird** [comment]
I think it's an unexpected and very good idea! In the case of Chrome, the top bar is white, so I didn't think of this problem. Thank you for the great idea and I'm also curious about other team members' opinions!
예상치 못한 아주 좋은 아이디어라고 생각합니다! 크롬의 경우 상단바 부분이 흰색이라서 이런 문제는 미처 생각하지 못했습니다. 좋은 아이디어 감사하며 다른 팀원들의 의견도 궁금합니다!

**Comment by bentshrimp** [comment]
I'll make it happen.

**Comment by sinji2102** [comment]
![image](https://github.com/AgainIoT/Open-Set-Go/assets/66528589/26d4d403-8c1f-440e-b63d-5145dd424cf0)

Unlike the colorful Google logo, our logo is monochrome, so I think the original logo is prettier if we use a different color theme.
알록달록한 구글 로고와 달리 우리의 로고는 단색이라 다른 색의 테마를 사용할 경우 원래 로고가 더 예쁜 것 같습니다.
